src/enums/index.ts

```diff
@@ -1,7 +1,7 @@
 export * from './account-type'
 export * from './advanced-search-tabs'
-export * from './company-type'
-export * from './entity-type'
+export * from './company-types'
+export * from './entity-types'
```

Comment:
First, I prefer enums to be plural (since they are a list of values -- arrays should be plural).

Second, in my next PR, I will have a couple of components named CompanyType and EntityType, so this resolves the name conflict.